README.md

```diff
@@ -21,26 +22,50 @@ Here are the steps for creating and opening a project:
 
 1. Go to [https://start.spring.io/](https://start.spring.io/).
 2. Select the project properties.
+   1. Select "Maven Project", as we will use Maven as the build tool.
+   2. Select "Java" as the language.
+   3. Select the most recent release version of Spring Boot. (Make sure it does
+      not have "SNAPSHOT" listed after it.)
+   4. Select the appropriate Java JDK version.
 3. Add dependencies.
+   1. Let's add the Spring Data JPA dependency to show how to add dependencies.
+   2. Click "ADD DEPENDENCIES".
+   3. Search for "spring data jpa".
+   4. Select "Spring Data JPA" from the list.
 4. Click on the “Generate” button on the bottom. This will download a zip file
    containing the SpringBoot project.
-5. Unzip the archive and open it in your preferred code editor or IDE.
+5. Unzip the archive and open it in a preferred code editor or IDE.
 
-![Spring Initializr home screen](https://curriculum-content.s3.amazonaws.com/java-spring-1/spring-initalizr.png)
+![Spring Initializr home screen](https://curriculum-content.s3.amazonaws.com/spring-mod-1/create-spring-boot-project/spring-initializr.png)
```

Comment:
Also, it looks like Spring Boot 3 will be released soon (November maybe). Should we consider using Spring Boot 3?